rb/lib/selenium/webdriver/common/keys.rb

```diff
@@ -98,7 +96,7 @@ module Keys
       #
 
       def self.[](key)
-        KEYS[key] or raise Error::UnsupportedOperationError, "no such key #{key.inspect}"
+        KEYS[key] || raise(Error::UnsupportedOperationError, "no such key #{key.inspect}")
```

Comment:
Why wrap raise with `()`. I know it's a Kernel method, but sort of treated as a keyword, which don't need parenthesis.